examples/Assistants/Example03_ListAssistantsWithPaginationAsync.cs

```diff
@@ -17,7 +18,8 @@ public async Task Example03_ListAssistantsWithPaginationAsync()
 
         int count = 0;
 
-        AsyncPageableCollection<Assistant> assistants = client.GetAssistantsAsync();
+        AsyncPageCollection<Assistant> assitantPages = client.GetAssistantsAsync();
```

Comment:
nit: stubborn `s` key (`assitantPages` -> `assistantPages`) -- you must have the same keyboard I do 😆

Reply:
TY!  It's the moving from left to right on the left hand motion ...